test/fixtures/change-code.js

```diff
@@ -77,7 +79,7 @@ function listener (code, signal) {
 function run (opt) {
   console.error(opt)
   var shell = process.platform === 'win32' ? null : { shell: '/bin/bash' }
-  exec(process.execPath + ' ' + __filename + ' ' + opt, shell, function (err, stdout, stderr) {
+  exec(join(process.execPath, ' ', __filename, ' ' + opt), shell, function (err, stdout, stderr) {
```

Comment:
This is a weird use of `path.join`.  Won't that result in something like `/path/to/node/` with a slash at the end?